rollup.config.js

```diff
@@ -0,0 +1,62 @@
+import nodeResolve from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import babel from "@rollup/plugin-babel";
+const createEsm = process.env.CREATE_ESM === "true";
+const createCjs = process.env.CREATE_CJS === "true";
+const createUmd = process.env.CREATE_UMD === "true";
+const input = "src/index.js"; // Update with your entry point
+const outputDir = "build"; // Update with your desired output directory
+const name = "Cytposcape";
+const configs = [];
+
+if (createEsm) {
+  configs.push({
+    input,
+    output: {
+      file: `${outputDir}/cytoscape.esm.js`,
```

Comment:
The file name ought to be the name of the extension, consistent with how the files are currently named.

Further, if a build directory is used, it should be called `dist` if it’s being used for published build artifacts.